wurst/systems/trade/GoldTrading.wurst

```diff
@@ -0,0 +1,50 @@
+package GoldTrading
+import LinkedList
+import ChatCommands
+import SimError
+import PlayerExtensions
+import StringExtensions
+import PlayerUtils
+
+CommandHandler giveGold = (player giver, LinkedList<string> args) -> begin
+    let tribeId = giver.getTribeId()
+    let playerId = findPlayerIdByStringMatchingTribe(args.get(1), tribeId)
+    let amount = args.get(2).toInt()
+    let currentGold = giver.getLumber()
+    let target = players[playerId]
+
+    if playerId < 0 or playerId > 11
+        simError(giver, "Command: -g player amount")
+    else if amount <= 0
+        simError(giver, "Amount must be greater than zero")

```

Comment:
Nitpicky but these should end with a period.

Reply:
K